### PR TITLE
resolves all bucklescript inferred name warning

### PR DESCRIPTION
**Pull request by idkjs**
This pull request resolves all the `inferred name` errors generated by `warning 105` from the compiler.

Example:
```sh
Warning 105: BuckleScript warning: fetch : the external name is inferred from val name is unsafe from refactoring when changing value name
```

### CODE REVIEW

File: src/apis/ActionSheetIOS.md [hunk 1]
```diff
@@ -18,10 +18,10 @@ external options:
     unit
   ) =>
   options =
-  "";
+  "options";
```
[the remark this answers is not in the document]
Comment (MoOx) [reply src/apis/ActionSheetIOS.md]:
```suggestion
  "";
```

File: src/apis/AsyncStorage.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ wip: true
 type error;
 
 [@bs.scope "AsyncStorage"] [@bs.module "react-native"]
-external getItem: string => Js.Promise.t(Js.Null.t(string)) = "";
+external getItem: string => Js.Promise.t(Js.Null.t(string)) = "getItem";
```
Comment (MoOx) [inline src/apis/AsyncStorage.md]: This file is not fully up to date with the .re

File: src/apis/BackHandler.re [hunk 3]
```diff
@@ -1,14 +1,14 @@
 type remove = {. [@bs.meth] "remove": unit => unit};
 
 [@bs.scope "BackHandler"] [@bs.module "react-native"]
-external exitApp: unit => unit = "";
+external exitApp: unit => unit = "exitApp";
```
Comment (MoOx) [inline src/apis/BackHandler.re]: similar update in .md file is missing here & in all others apis below (but maybe: should we care? no big deal)

File: src/apis/NativeModules.re [hunk 4]
```diff
@@ -1,12 +1,12 @@
 // We recommend that you bind your own custom native modules like this:
 //
 // [@bs.module "react-native"] [@bs.scope ("NativeModules", "MyCustomModule")]
-// external myFunc: unit => unit = "";
+// external myFunc: unit => unit
```
Comment (MoOx) [inline src/apis/NativeModules.re]:
```suggestion
// external myFunc: unit => unit = "myFunc"
```

**Comment by idkjs** [comment]
There were so many that I filtered out the `.md` to handle the reason files. Then forgot to go back. And do the `.md`. That is why some are done and some not. Give me a second to fix. Done now.

File: src/apis/BackHandler.re (hunk 3 above)
Comment (idkjs) [reply src/apis/BackHandler.re]: Done

File: src/apis/ActionSheetIOS.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (idkjs) [reply src/apis/ActionSheetIOS.md]: Done

File: src/apis/AsyncStorage.md (hunk 2 above)
Comment (idkjs) [reply src/apis/AsyncStorage.md]: Done

**Pull request merged**